semantics.json

```diff
@@ -174,6 +174,24 @@
                         "label": "License"
                       }
                     ]
+                  },
+                  {
+                    "name": "nextPreviousSlide",
+                    "type": "select",
+                    "label": "Go To Next or Previous Slide",
+                    "description": "This will override the Go To Slide Number",
+                    "importance": "low",
+                    "optional": true,
+                    "options": [
+                      {
+                        "value": "next",
+                        "label": "Next"
+                      },
+                      {
+                        "value": "previous",
+                        "label": "Previous"
+                      }
```

Comment:
Good idea. I added a [new commit](https://github.com/h5p/h5p-course-presentation/pull/40/commits/0cf2d3c48e2e0a16b4500d618b083722a3e31142) for this using your suggestions.